EnglishFilter/sections/foreign.txt

```diff
@@ -4949,31 +4945,13 @@ ohli24.net###exoall
 ! https://github.com/AdguardTeam/AdguardFilters/issues/84316
 sutv7.com##.basic-banner
 ||sutv7.com/data/apms/background/$image
-! https://github.com/AdguardTeam/AdguardFilters/issues/83065
-tgd.kr###header-top-notice
 ! https://github.com/AdguardTeam/AdguardFilters/issues/81162
 ||ams.rmcnmv.naver.com/item/create/*?ams_ctgr=$xmlhttprequest,domain=sports.news.naver.com
 ! g-enews.com ads
 g-enews.com##.mad
 ! https://github.com/AdguardTeam/AdguardFilters/issues/80447
 ||data.news.zumst.com/alyac/popular/news.json
 121.189.38.123^$network
-! https://github.com/AdguardTeam/AdguardFilters/issues/81240
-@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=ppss.kr
-ppss.kr#$?#.ppss-title-banner { remove: true; }
-ppss.kr#@#main#genesis-content>div:not(.archive-description):not(.archive-pagination):not(.fb-quote):not(.home-top):not(.widget-area):not([id^="dablewidget_"])
-ppss.kr#@#main#genesis-content>span
-@@||ad.doubleclick.net/ddm/adj^$xmlhttprequest,domain=ppss.kr
-ppss.kr#@#.adsbygoogle
-ppss.kr#@##ad-unit
-ppss.kr#@#.ad-unit
-@@||ppss.kr^$generichide
-!#if (adguard_app_windows || adguard_app_mac || adguard_app_android)
-||ppss.kr^$replace=/(\]\(\{'mode':[\w]+\[[\w]+\([\w]+\)\]\}\);\}\))\.append\([\w]+\)(;})/\$1\$2/,~script,~xmlhttprequest,~stylesheet
-||ppss.kr^$replace=/(;\}\)\,)[\w]+\.append\([\w]+\)\,([\s\S]*?;\}\()/\$1\$2/,~script,~xmlhttprequest,~stylesheet
-||ppss.kr^$replace=/(;\}\))\['append'\]\([\s\S]*?\);(\})/\$1\$2/,~script,~xmlhttprequest,~stylesheet
-||ppss.kr^$replace=/((?:throw )?new Error)[\s\S]*?;/null;/,~script,~xmlhttprequest,~stylesheet
```

Comment:
Are `$replace` rules also not required?

Reply:
Yes.
If you want to verify it, please disable AdGuard Base filter and enable List-KR.

Reply:
Ok.